test/mocha.opts

```diff
@@ -1,4 +1,3 @@
 --compilers js:babel-register
 --require test/helpers/globals
 --require babel-polyfill
---reporter mocha-trumpet-reporter
```

Comment:
That and the ASCII art makes it hard to see the error + debug info. I always use the `min` reporter as it shows nothing but debugging information.